### PR TITLE
User dashboard

### PR DESCRIPTION
**Pull request by honorine22**
#### What does this PR do?
 Implement dashboard skeleton and user dashboard
 ...

#### Description of  the task to be completed?
user dashboard for an admin and normal user
...

#### How should this be manually tested?
Wrap your component in DashboardLayout
http://localhost:5173/dashboard
...

#### Any background context you want to provide?
N/A
...

#### What are the relevant pivotal tracker stories?

...

#### Screenshots (if appropriate)

...

### CODE REVIEW

**Review by dushimsam — APPROVED** [review]
LGTM

File: coverage/clover.xml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        <line num="78" count="0" type="stmt"/>
+        <line num="82" count="0" type="cond" truecount="0" falsecount="1"/>
+        <line num="83" count="0" type="stmt"/>
+        <line num="85" count="0" type="stmt"/>
+        <line num="86" count="0" type="cond" truecount="0" falsecount="1"/>
+        <line num="87" count="0" type="stmt"/>
+      </file>
+    </package>
+  </project>
+</coverage>
```
Comment (Nkbtemmy) [inline coverage/clover.xml]: This coverage folder doesn't need to be pushed

File: coverage/lcov-report/base.css [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+.wrapper {
+  min-height: 100%;
+  height: auto !important;
+  height: 100%;
+  margin: 0 auto -48px;
+}
+.footer, .push {
+  height: 48px;
+}
```
Comment (Nkbtemmy) [inline coverage/lcov-report/base.css]: This one also doesn't need to be pushed

File: coverage/lcov-report/block-navigation.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                break;
+            case 66: // b
+            case 75: // k
+            case 80: // p
+                goToPrevious();
+                break;
+        }
+    };
+})();
+window.addEventListener('keydown', jumpToCode);
```
Comment (Nkbtemmy) [inline coverage/lcov-report/block-navigation.js]: Please put coverage folder in `.gitignore` file

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]

File: src/plugins/configUrl.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+export const backendUrl = 'http://localhost:3000/api'// Replace with your actual backend URL
```
Comment (Nkbtemmy) [inline src/plugins/configUrl.ts]: Please use the backend URL which is in .env!!

`export const backendUrl =  import.meta.env.VITE_BASE_URL;`

File: coverage/lcov-report/block-navigation.js (hunk 3 above)
Comment (honorine22) [reply coverage/lcov-report/block-navigation.js]: Sure. It's already there.

**Review by Nkbtemmy — APPROVED** [review]

File: src/plugins/configUrl.ts (hunk 4 above)
Comment (honorine22) [reply src/plugins/configUrl.ts]: Sure. I have changed it.

**Pull request merged**